Update RnLibLog.sys

RobotSigmund opened the pull request:
Closes #21

Added comments.
Added proper variable naming.
Simplified functionality.
Added errormessages, easier to debug during config.

RobotSigmund committed:
Update RnLibLog.sys

RobotSigmund committed:
Update RnLibLog.sys

RobotSigmund committed:
Update and rename RnLibLog.sys to RNL_B_Log.sys

TheHarvard reviewed (changes requested):
Have added comments to specific parts of code.

in summary:

there is some deviation from the specification in #21.

A couple of suggestions for changes.

Have not tested the module in a virtual robot yet. 

This is one of the first review processes, let's go over it in the meeting today!

RobotSigmund committed:
Rename RnLibLog.md to RnLibLog.markdown

RobotSigmund committed:
Rename RnLibLog.markdown to RNL_B_Log.sys.markdown

RobotSigmund committed:
Update RNL_B_Log.sys

RobotSigmund committed:
Update RNL_B_Log.sys

RobotSigmund committed:
Update RNL_B_Log.sys

RobotSigmund committed:
Update RNL_B_Log.sys

Added function for archive size:
logFile_getArchiveSize()

RobotSigmund committed:
Update RNL_B_Log.sys.markdown

RobotSigmund committed:
Update RNL_B_Log.sys

Fixes a bug; If logfile doesn't exist, it will not be created with timestamp. The next write will move the previous one to the archive. And the next day the "first" line will be overwritten and lost.
Also added taskname to be written on each line.

RobotSigmund committed:
Create RNL_B_Log_TestMod.sys

Temporary file for testing module functionality. Should this be in the referenced module file?

RobotSigmund reviewed (commented):
Change request ack.
